tests/test_container_tasks.py

```diff
@@ -73,3 +81,78 @@ def test_get_latest_version_containerized(monkeypatch):
         "conda-smithy", attrs, all_version_sources()
     )
     assert data["new_version"] == conda_smithy.__version__
+
+
+def test_container_tasks_parse_feedstock(monkeypatch):
+    if "CI" not in os.environ:
+        monkeypatch.setenv("CI", "true", prepend=False)
+
+    res = subprocess.run(
+        [
+            "docker",
+            "run",
+            "--rm",
+            "-t",
+            f"{get_default_container_name()}",
+            "python",
+            "/opt/autotick-bot/docker/run_bot_task.py",
+            "parse-feedstock",
+            "--existing-feedstock-node-attrs=conda-smithy",
+        ],
+        capture_output=True,
+    )
+    assert res.returncode == 0
+    data = json.loads(res.stdout.decode("utf-8"))
+    with lazy_json_override_backends(["github"], use_file_cache=False), LazyJson(
+        "node_attrs/conda-smithy.json"
+    ) as lzj:
+        attrs = copy.deepcopy(lzj.data)
+
+    assert data["feedstock_name"] == attrs["feedstock_name"]
+    assert not data["parsing_error"]
+    assert data["raw_meta_yaml"] == attrs["raw_meta_yaml"]
+
+
+def test_container_tasks_parse_feedstock_json(monkeypatch):
+    if "CI" not in os.environ:
+        monkeypatch.setenv("CI", "true", prepend=False)
+
+    with lazy_json_override_backends(["github"], use_file_cache=False):
+        with LazyJson("node_attrs/conda-smithy.json") as lzj:
+            attrs = copy.deepcopy(lzj.data)
+            existing_feedstock_node_attrs = dumps(lzj.data)
+
+    res = subprocess.run(
+        [
+            "docker",
+            "run",
+            "--rm",
+            "-t",
+            f"{get_default_container_name()}",
```

Comment:
```suggestion
            get_default_container_name(),
```

Given how often this type of subprocess is run, I wonder if a helper utility function would help avoid the duplication.